ds-caselaw-ingester/lambda_function.py

```diff
@@ -544,11 +576,17 @@ def handler(event, context):
                 s3_client,
             )
 
-    if api_client.get_published(uri):
+    force_publish = (
+        metadata.get("parameters", {})
+        .get("INGESTER_OPTIONS", {})
```

Comment:
where is this set? in the s3 messages?

Reply:
could we add tests for these new scenarios:

- `auto_publish`= True, api_client.get_published(uri) = False
-`auto_publish`= True, api_client.get_published(uri) = True

Reply:
`metadata` is the json blob from the TDR-2023-XXX-metadata.json file in the tar.gz